benchmark/_cli.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+// Create an object of all benchmark scripts
+const benchmarks = {};
+fs.readdirSync(__dirname)
+  .filter(function(name) {
+    return fs.statSync(path.resolve(__dirname, name)).isDirectory();
+  })
+  .forEach(function(category) {
+    benchmarks[category] = fs.readdirSync(path.resolve(__dirname, category))
+      .filter((filename) => filename[0] !== '.' && filename[0] !== '_');
+  });
+
+function CLI(usage, settings) {
+  if (!(this instanceof CLI)) return new CLI(usage, settings);
+
+  if (process.argv.length < 3) {
+    this.abort(usage);
+    return;
```

Comment:
`.abort()` runs `process.exit(1)`. no need for the `return`. unless you're using it as a visual queue or something.